Makefile

```diff
@@ -8,9 +8,10 @@
 
 # To update the Teleport version, update VERSION variable:
 # Naming convention:
-#	for stable releases we use "1.0.0" format
-#   for pre-releases, we use   "1.0.0-beta.2" format
-VERSION=4.4.0-alpha.1
+#   Stable releases:   "1.0.0"
+#   Pre-releases:      "1.0.0-beta.2"
```

Comment:
`beta` or `alpha`?

Reply:
Both, plus `rc`... didn’t want to add multiple examples, but we could.